package.json

```diff
@@ -4,7 +4,9 @@
   "description": "openaps oref0 reference implementation of the reference design",
   "scripts": {
     "test": "make test",
-    "global-install": "npm install && sudo npm install -g && sudo npm link && sudo npm link oref0"
+    "deprecated-global-install" : "npm install && sudo npm install -g && sudo npm link && sudo npm link oref0",
+    "global-install": "Error: global-install is deprecated. Please use 'npm install -g oref0' for a packaged version, or 'npm install -g .' for a git checkout. See https://github.com/openaps/oref0/pull/302",
```

Comment:
@scottleibrand 
I don't see what you mean. I deprecated the `global-install` to `deprecated-global-install`. 
Currently autotune branch always installs oref@0.3.3 with `oref0-setup.sh` while currently the branch is at 0.3.5-dev and so it would not install oref0-autotune in `/usr/local/bin`.
When I use `npm install -g` it will install and link the new files (e.g. `/usr/local/bin/oref0-autotune`. So I think we should stop using non stantard install procedures like `global-install`

See also: https://github.com/PieterGit/oref0/blob/oref0_version/bin/oref0-setup.sh#L247-L266